crates/nu-utils/standard_library/tests.nu

```diff
@@ -1,135 +1,14 @@
-use std.nu
-
-def test_assert [] {
-    def test_failing [code: closure] {
-        let code_did_run = (try { do $code; true } catch { false })
-
-        if $code_did_run {
-            error make {msg: (view source $code)}
-        }
-    }
-
-    std assert true
-    std assert (1 + 2 == 3)
-    test_failing { std assert false }
-    test_failing { std assert (1 + 2 == 4) }
-
-    std assert eq (1 + 2) 3
-    test_failing { std assert eq 1 "foo" }
-    test_failing { std assert eq (1 + 2) 4) }
-
-    std assert ne (1 + 2) 4
-    test_failing { std assert ne 1 "foo" }
-    test_failing { std assert ne (1 + 2) 3) }
-}
-
-def tests [] {
-    use std.nu assert
-
-    let branches = {
-        1: { -1 }
-        2: { -2 }
-    }
-
-    assert ((std match 1 $branches) == -1)
-    assert ((std match 2 $branches) == -2)
-    assert ((std match 3 $branches) == $nothing)
-
-    assert ((std match 1 $branches { 0 }) == -1)
-    assert ((std match 2 $branches { 0 }) == -2)
-    assert ((std match 3 $branches { 0 }) == 0)
-}
-
-def test_path_add [] {
-    use std.nu "assert eq"
-
-    with-env [PATH []] {
-        assert eq $env.PATH []
-
-        std path add "/foo/"
-        assert eq $env.PATH ["/foo/"]
-
-        std path add "/bar/" "/baz/"
-        assert eq $env.PATH ["/bar/", "/baz/", "/foo/"]
-
-        let-env PATH = []
-
-        std path add "foo"
-        std path add "bar" "baz" --append
-        assert eq $env.PATH ["foo", "bar", "baz"]
-
-        assert eq (std path add "fooooo" --ret) ["fooooo", "foo", "bar", "baz"]
-        assert eq $env.PATH ["fooooo", "foo", "bar", "baz"]
-    }
-}
-
-
-def test_dirs [] {
-
-    def "myassert" [
-        predicate: bool
-        msg?:string = "..."
-        --verbose = false (-v)  # enable to see successful tests
-    ] {
-        if not $predicate {
-            let span = (metadata $predicate).span
-            error make {msg: $"Assertion failed checking ($msg)",
-                        label: {text: "Condition not true" start: $span.start end: $span.end}}
-        } else {
-            if $verbose {
-                echo $"check succeeded: ($msg)"
-            }
+def main [] {
+    for test_file in (ls $"($env.FILE_PWD)/test_*.nu" -f | get name) {
+        let $module_name = ($test_file | path parse).stem
+        echo $"(ansi yellow)INFO Run tests in ($module_name)(ansi reset)"
```

Comment:
i would add a newline just before this `echo` line, to separate logical blocks of this `for` loop :yum: